helm/ph-ee-engine/values.yaml

```diff
@@ -897,3 +903,55 @@ keycloak:
     rules:
       - host: 'keycloak.mifos.io'
     tls: []
+
+tupande_connector:
+  enabled: true
+  replicas: 1
+  image: "oaftech.azurecr.io/oneacrefund/tupande-connector"
+  imageTag: "0.0.1"
+  imagePullPolicy: "Always"
+  LOGGING_LEVEL_ROOT: "INFO"
+  SPRING_PROFILES_ACTIVE: ""
+  containerPort: 5000
+  limits:
+    memory: "512M"
+    cpu: "500m"
```

Comment:
This value is too high. Prefer 200m
And you are locking the memory to 512M prefer 1Gi

Reply:
ok cool